src/site/content/en/fast/signed-exchanges/index.md

```diff
@@ -0,0 +1,326 @@
+---
+layout: post
+title: Signed Exchanges (SXGs) overview
```

Comment:
```suggestion
title: Signed Exchanges (SXGs)
```
I've changed my mind on this. If we're going to have conceptual and how-to content in a single page, then we should leave the name more generic. Because I would never think to look for how-to content in a page called "Signed Exchanges overview".